database/bigMenuData.js

```diff
@@ -2,67 +2,68 @@ const faker = require('faker');
 const fs = require('file-system');
 const path = require('path');
 const csvWriter = require('csv-write-stream');
-const headers =  {
-  headers: [
-    'id',
-    'mealType',
-    'Item',
-    'Des',
-    'Price'
-  ]
+const headers = {
+  headers: ['Breakfast', 'Lunch', 'Dinner', 'Brunch', 'HappyHour']
 }
 const writer = csvWriter(headers);
 
+  var generateMenuObject_100000_helper = function() {
+    result = {};
+    var result = {'Breakfast':[], 'Lunch':[], 'Dinner':[], 'Brunch': [], 'HappyHour':[]};
+    var length = faker.random.number({min: 1, max: 3}) ;
 
+    for ( type of headers.headers ) {
+      for (var i = 0; i < length ; i++) {
+        result[type].push(JSON.stringify ({   itemName: faker.lorem.word(), itemDescription: faker.lorem.paragraph(), itemPrice: "$" + faker.commerce.price()} ));
+      }
+    }
+    return result;
+  };
 
-generateMenuObject = function(max, file ,callback) {
+ var generateMenuObject_100000 = function(max, file ,callback) {
```

Comment:
I wonder if it might be more optimal to merge this generateMenuObject function with its corresponding helper function?